charts/neon-proxy/templates/deployment.yaml

```diff
@@ -68,6 +68,18 @@ spec:
             - {{ . }}
             {{- end }}
             {{- end }}
+            {{- if .Values.settings.domain }}
+            - --tls-key
+            - /certs/tls.key
+            - --tls-cert
+            - /certs/tls.crt
+            {{ end }}
+          {{- if .Values.settings.domain }}
```

Comment:
you adding the same condition again, we can remove these 2 lines (end and next if), but this ok as well.

Reply:
I would strongly oppose having if and end with different indentation.